docs/eval/2026-06-21-w35-class-c-freshness-measured.md

```
@@ -0,0 +1,106 @@
+| Model / budget | `haiku`, `--max-budget-usd 0.50` per session (hard per-session cap) |
+| Scenarios | `crates/rb-eval/scorecard/memory_scorecard_scenarios.json` (4 freshness) |
+| Raw data | [`2026-06-21-w35-class-c-freshness-n10.tsv`](./2026-06-21-w35-class-c-freshness-n10.tsv) (columns: `dimension  scenario  arm  run  success  turns  mie`) |
+
+Validation also ran the smokes the build sequence calls for, all green: the local
+`--self-test` (judge + aggregation math, no API); a GitHub-dispatch **N=1**
+plumbing smoke ([run 27917826435](https://github.com/brianluby/rusty-brain/actions/runs/27917826435));
+and a local live N=1.
+
+## Result (N=10)
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)